bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import discord
+import requests
+from discord.ext import commands
+
+from bot.constants import Tokens
+
+log = logging.getLogger(__name__)
+
+UnClient_id = Tokens.unsplash_key
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def init(self, bot: commands.Bot) -> None:
+        """Init function."""
+        self.bot = bot
+        self.current_channel = None
+
+    @commands.command(aliases=["earth"])
+    async def earth_photos(self, ctx: commands.Context) -> None:
+        """Returns a random photo of earth, sourced from Unsplash."""
+        photorequest = requests.get("https://api.unsplash.com/photos/random?query=earth&client_id=" + UnClient_id)
+        photojson = photorequest.json()
+        photourls = photojson.get('urls')
+        urltosend = photourls.get('regular')
+        userjson = photojson.get('user')
+        username = userjson.get('name')
+        embed = discord.Embed(title="Earth Photo", description="A photo of Earth from Unsplash.", color=0x66ff00)
```

Comment:
Can we break this down into lines for easier readability, for example like this:
```py
embed = discord.Embed(
     title=random.choice(ERROR_REPLIES),
     color=Colours.soft_red,
     description="You don't have permission to view this channel."
 )
```